src/Common/tests/Performance/PerfUtils.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.IO;
+
+namespace System
+{
+    /// <summary>Static helper class for performance tests</summary>
+    public static class PerfUtils
+    {
+        private static Random rand = new Random(1234132);
```

Comment:
Benchmarks are not run concurrently.  But there's no guarantee about the *order* in which they run, so if this is shared among several tests they may get different results depending on ordering.